tests/components/airthings/test_config_flow.py

```diff
@@ -23,17 +24,17 @@
     DhcpServiceInfo(
         hostname="airthings-view",
         ip="192.168.1.100",
-        macaddress="00:00:00:00:00:00",
+        macaddress=dr.format_mac("00:00:00:00:00:00").replace(":", ""),
```

Comment:
Wait, why call this and not just enter the right format directly into the mock?

Reply:
> Wait, why call this and not just enter the right format directly into the mock?

I think clarity of intent is sometimes more important than performance, but I've adjusted it